### DISCUSSION TITLE
‘--moduleResolution' option must be: 'node', 'classic', 'node12',  'nodenext'

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#1066] [Q&A]
Hi, 
    I migrate to version 1.1.0-RC10 from 1.0.0, my project is a theia 1.39.0 based application, when I build I get an error about '--moduleResolution' in @eclipse-glsp/config/ts-config
![image](https://github.com/eclipse-glsp/glsp/assets/4537809/4f7e1808-f9d8-469d-9875-7c2fb0fc63f9)
 '--moduleResolution' is 'node' in version 1.0.0. now it is changed to 'Node16'
![image](https://github.com/eclipse-glsp/glsp/assets/4537809/3ecaf4ab-6117-477c-b53e-2904d2846a67)

**Comment by tortmayr** [comment]
Hi @wss29,
looks like this error is related to either an incompatible node or typescript version.
The 1.1.0-RC10 release candiate requires
- node >=16.11.0
- typescript =>4.7

Disclaimer: The Release Candiates contain lots of breaking changes that are currently not documented so migrating from 1.0.0 to the latest RC might be not that trivial. If there is no urgency I would recommend to postpone the migration until the end of August. Then the new major release (including a migration guide) will be available.

**Comment by rsoika** [comment]
Hi, @tortmayr  could it be that I am also affected from breaking changes when I try to run my Diagram Module which is based on 1.1.0-RC04 in my new Eclipse Integration project which is based on 1.1.0-RC10?

**Reply by tortmayr** [reply]
Yes, that could be the case.
However, we have never encountered the error that you get in the Eclipse Integration. API breaks should typically result in compilation errors. Since that is not the case for you I would rather suspect that there is another issue specific to your project setup.
